xload/type/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package xloadtype contains commonly used types for working with xload.Loader.
+package xloadtype
```

Comment:
What do you think of making this a top level `types`. Then these can be useful outside of xload

Reply:
Don't have a strong opinion but I have not encountered any need outside value parsing for such types.

Reply:
Also, `types` would be a very common package name. Can do `xtooltypes` or similar but I do not see any immediate usage. We can maybe move these later, if needed?